environment.yml

```diff
@@ -1,21 +1,337 @@
 name: jump_sc
 channels:
   - conda-forge
+  - defaults
 dependencies:
-  - conda-forge::python
-  - conda-forge::boto3
-  - conda-forge::jupyter
-  - conda-forge::jupyterlab
-  - conda-forge::ipykernel
-  - conda-forge::nb_conda_kernels
-  - conda-forge::jupyter_contrib_nbextensions
-  - conda-forge::pandas
-  - conda-forge::scipy
-  - conda-forge::numpy
-  - conda-forge::scikit-learn
-  - conda-forge::plotnine
-  - conda-forge::umap-learn
-  - conda-forge::matplotlib
+  - _libgcc_mutex=0.1=conda_forge
```

Comment:
this is an incredibly strict conda environment that is not likely to work on any computer other than your own.

I recommend using a simplified conda environment, which specifies only essential packages and versions.

If you would like to go the very specific environment route, then you should use docker.

Reply:
That is good to know, I always just exported my conda environment to a yaml file. After talking with Mike and Jenna, I think I will manually specify the packages I install in the conda environment file. This isn't a perfect solution, so I'm open to suggestions.

Reply:
manually specifying the packages is the way to go. It's a trade-off between rigid environments that don't work on many computers and loose environments that fail to reproduce the analysis. I recommend moving a bit towards the loose environment.